h2o-docs/src/product/getting-started/getting-started.rst

```diff
@@ -0,0 +1,194 @@
+Getting started
+===============
+
+Here are some helpful links to help you get started learning H2O.
+
+Downloads page
+--------------
+
+To begin, download a copy of H2O from the `Downloads page <https://h2o.ai/resources/download/>`__.
+
+1. Click H2O Open Source Platform or scroll down to the H2O section. Here you have access to the different ways to download H2O:
```

Comment:
I am surprised we called our product just H2O and not H2O-3. :thinking: I think it collides with the company name a little bit and can be confusing.

Reply:
I think it might be because of how old the guide is. I can update to H2O-3 as I go through now, though, to be less confusing if we think that's best?

Reply:
It would be great. But we should check it with @wendycwong and @mmalohlava. On our website, the product is also called H2O.

Reply:
@hannah-tillman :  I check the resource page and h2o-3 is used to denote h2o opensource:

![Screenshot 2024-05-22 at 11 25 58 AM](https://github.com/h2oai/h2o-3/assets/7231712/69d9a2a7-aa38-432e-9113-ba7872fbdf0b)

Reply:
So, we either use H2O open source or just use H2O-3.  I suggest using H2O-3 after setting H2O open source == H2O-3.